.github/workflows/main.yml

```diff
@@ -29,6 +29,10 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install coveralls setuptools tox wheel
 
+    - name: Install mock (For Python < 3.3)
+      if: ${{ matrix.python_version == '2.7' }}
+      run: "python -m pip install mock"
```

Comment:
Thanks for making the changes! I'll have a more detailed look later, but it seems that this mock code can go away and that there's still a merge conflict.

Reply:
Ah right. Forgot about that. I currently don't get any merge conflicts. After merging your `master` into my feature branch yesterday I fixed all the conflicts and now I don't see anything when merging `upstream/master` into my `master`. Might still be something in the other direction though (even though GitHub does not show any conflicts either :thinking:).

I will try to rebase later today. If it's not getting too gnarly I can force-push after that giving a cleaner history. In the same go I can drop the py2/3 fixes as long as my commits were atomic enough :)

What's your thought on the rebase?

Reply:
My bad, I was just confused by the GitHub UI. No need to rebase.

Reply:
Before merging, I'll squash your changes into one commit, so you don't need to clean up the history.